docs/patterns/container-patterns/docker-builds/alpine-simple-api.md

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_position: 1
+title: Alpine Image
+description: Building a Rust project to be deployed on an Alpine image
+keywords: [rust,docker,container,containers,alpine]
+---
+
+## Introduction
+
+The Alpine page on Dockerhub describes itself like this:
+
+> Alpine Linux is a Linux distribution built around musl libc and BusyBox. The image is only 5 MB in size and has access to a package repository that is much more complete than other BusyBox based images. This makes Alpine Linux a great image base for utilities and even production applications. Read more about Alpine Linux here and you can see how their mantra fits in right at home with Docker images. -- Alpine
+
+What that means for a developer is that you are running your code on top of a slim and secure Linux base image.  By using something this minimal, you can reduce your vunerability footprint and also reduce the time to launch in container orchestrators such as Elastic Container Service (ECS) and AppRunner. 
+
+## Sample Solution
+
+A template for this pattern can be found under the [./templates](https://github.com/serverlessdevelopers/serverless-rust/tree/main/templates/patterns/container-patterns/alpine-simple-api/) directory in the GitHub repo. You can use the template to get started with your own project.  The Dockerfile included might be all you are after, and that's OK.  But you can also use the sample API and the Dockerfile to experiment with different values.  There are also some release configuration settings in the Cargo.toml that enhance the final binary generated by Cargo.
```

Comment:
You mention release configuration settings in Cargo.toml... Could you explain them, or add them to the cargo.toml page?

Reply:
Ok, just seen the next paragraph. Maybe mention it's going to be discussed further down